Library/Formula/rcssserver.rb

```diff
@@ -67,7 +67,31 @@ def install
 
     resources.each do |r|
       r.stage do
-        system "./bootstrap" if build.head?
+        Patch.create(:p1, <<EOD).apply
```

Comment:
Please submit this patch upstream somewhere and use that link or file it with [our patch repo](https://github.com/Homebrew/patches) after submitting upstream and use that instead of writing it inline in the formula file, Thanks!

Also need a brief in-formula explanation of why the patch is necessary and where it has been reported.